README.md

```diff
@@ -1,64 +1,50 @@
 ---
 page_type: sample
 languages:
-- csharp
+- python

```

Comment:
This section looks weird

![image](https://user-images.githubusercontent.com/39923391/75477473-40a50100-5951-11ea-8c4c-63b2faa3d505.png)